tests/auth_tests/test_views.py

```diff
@@ -770,6 +770,12 @@ def test_14377(self):
         response = self.client.get('/logout/')
         self.assertIn('site', response.context)
 
+    def test_25490(self):
+        # Bug 25490
```

Comment:
As above, use a more descriptive docstring like "The logout view should send "no-cache" headers for reasons described in #25490."